docs/docsite/rst/porting_guides/porting_guide_2.10.rst

```diff
@@ -1,23 +1,26 @@
+.. THIS DOCUMENT IS AUTOMATICALLY GENERATED BY ANTSIBULL! PLEASE DO NOT EDIT MANUALLY! (YOU PROBABLY WANT TO EDIT porting_guide_base_2.10.rst)
```

Comment:
I thought this is a comment since it starts with `.. `. Ah, but I just see, I forgot a newline :) https://stackoverflow.com/questions/4783814/how-to-comment-a-string-in-restructured-text